modules/service/src/main/scala/lucuma/odb/data/DatasetReference.scala

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
Will this get moved to lucuma-core, too, so that it can be used by the clients?

Reply:
Yes.  I believe I'll do that today in fact.

Reply:
https://github.com/gemini-hlsw/lucuma-core/pull/871